source/Octopus.Tentacle.Tests.Integration/Support/ScriptServiceVersion.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Octopus.Tentacle.Tests.Integration.Support
+{
+    public enum ScriptServiceVersionToTest
+    {
+        None,
```

Comment:
nit: will none make the caller think that no script service will be tested? Maybe `NotSet` etc.

Reply:
yeah, fair call

Reply:
@nathanwoctopusdeploy I changed this in [193b4df](https://github.com/OctopusDeploy/OctopusTentacle/pull/678/commits/193b4df0808c86d4cb807f259d30bd65bf747f58) so that if you just want to test the versions supported by the tentacle, then the value is now `TentacleSupported`